### PR TITLE
Refactor structure

### PR DESCRIPTION
**Pull request by Guillermo849**

### CODE REVIEW

File: service/runner.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require_relative 'caesar_cipher'
+require_relative 'file_operations/file_reader'
+require_relative 'file_operations/file_writer'
+
+class Runner
+  def Runner.run
```
Comment (OlekKovalevych) [inline service/runner.rb:8]:
```suggestion
  def self.run
```

Comment (OlekKovalevych) [reply service/runner.rb:8]: DOCS for ''self'' keyword https://www.scaler.com/topics/ruby-class-self/

**Review by OlekKovalevych — APPROVED** [review]

**Comment by OlekKovalevych** [comment]
LGTM 🥇

File: service/runner.rb (hunk 1 above)
Comment (Guillermo849) [reply service/runner.rb]: +1

**Pull request merged**